sdk/src/main/java/com/bugsnag/android/ConfigFactory.java

```diff
@@ -100,7 +102,8 @@ static void populateConfigFromManifest(@NonNull Configuration config,
         config.setSendThreads(data.getBoolean(MF_SEND_THREADS, true));
         config.setPersistUserBetweenSessions(
             data.getBoolean(MF_PERSIST_USER_BETWEEN_SESSIONS, false));
-
+        config.setDetectNdkCrashes(data.getBoolean(MF_DETECT_NDK_CRASHES, false));
+        config.setDetectAnrs(data.getBoolean(MF_DETECT_ANRS, false));
```

Comment:
We should only set the field when the key is present in the manifest, otherwise we will have to alter code in two places when the default behaviour for `detectNdkCrashes` changes to `true`.